Description: Reverts the Maven artifacts using the jakarta namespace to javax to preserve the backward compatibility.
 The new jakarta namespace will be used with the version 2.0 of JavaMail.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed


--- a/mail/pom.xml
+++ b/mail/pom.xml
@@ -28,16 +28,16 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.sun.mail</groupId>
-    <artifactId>jakarta.mail</artifactId>
+    <artifactId>javax.mail</artifactId>
     <packaging>jar</packaging>
     <name>Jakarta Mail API</name>
 
     <properties>
 	<mail.extensionName>
-	    jakarta.mail
+	    javax.mail
 	</mail.extensionName>
 	<mail.moduleName>
-	    jakarta.mail
+	    javax.mail
 	</mail.moduleName>
 	<mail.specificationTitle>
 	    Jakarta Mail API Design Specification
--- a/pom.xml
+++ b/pom.xml
@@ -659,7 +659,7 @@
 			<additionalDependencies>
 			    <additionalDependency>
 				<groupId>com.sun.mail</groupId>
-				<artifactId>jakarta.mail</artifactId>
+				<artifactId>javax.mail</artifactId>
 				<version>${mail.version}</version>
 			    </additionalDependency>
 <!--
@@ -712,7 +712,7 @@
 	    </dependency>
 	    <dependency>
 		<groupId>com.sun.mail</groupId>
-		<artifactId>jakarta.mail</artifactId>
+		<artifactId>javax.mail</artifactId>
 		<version>${mail.version}</version>
 	    </dependency>
 	    <dependency>
--- a/mailapi/pom.xml
+++ b/mailapi/pom.xml
@@ -48,7 +48,7 @@
 	    com.sun.mail.handlers; version=${mail.osgiversion}
 	</mail.packages.export>
 	<mail.bundle.symbolicName>
-	    jakarta.mail.api
+	    javax.mail.api
 	</mail.bundle.symbolicName>
 	<mail.extraClasses>javax/**</mail.extraClasses>
     </properties>
@@ -113,7 +113,7 @@
 			    <artifactItems>
 				<artifactItem>
 				    <groupId>com.sun.mail</groupId>
-				    <artifactId>jakarta.mail</artifactId>
+				    <artifactId>javax.mail</artifactId>
 				    <version>${mail.version}</version>
 				    <classifier>sources</classifier>
 				    <outputDirectory>
@@ -128,7 +128,7 @@
 		    <artifactItems>
 			<artifactItem>
 			    <groupId>com.sun.mail</groupId>
-			    <artifactId>jakarta.mail</artifactId>
+			    <artifactId>javax.mail</artifactId>
 			    <version>${mail.version}</version>
 			</artifactItem>
 		    </artifactItems>
--- a/mailapijar/pom.xml
+++ b/mailapijar/pom.xml
@@ -34,20 +34,20 @@
 	<version>1.6.5</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>jakarta.mail</groupId>
-    <artifactId>jakarta.mail-api</artifactId>
+    <groupId>javax.mail</groupId>
+    <artifactId>javax.mail-api</artifactId>
     <packaging>jar</packaging>
     <name>Jakarta Mail API jar</name>
 
     <properties>
 	<mail.extensionName>
-	    jakarta.mail
+	    javax.mail
 	</mail.extensionName>
 	<mail.packages.export>
 	    javax.mail.*; version=${mail.spec.version}
 	</mail.packages.export>
 	<mail.bundle.symbolicName>
-	    jakarta.mail-api
+	    javax.mail-api
 	</mail.bundle.symbolicName>
     </properties>
 
@@ -75,7 +75,7 @@
 			    <artifactItems>
 				<artifactItem>
 				    <groupId>com.sun.mail</groupId>
-				    <artifactId>jakarta.mail</artifactId>
+				    <artifactId>javax.mail</artifactId>
 				    <version>${mail.version}</version>
 				    <classifier>sources</classifier>
 				    <outputDirectory>
@@ -90,7 +90,7 @@
 		    <artifactItems>
 			<artifactItem>
 			    <groupId>com.sun.mail</groupId>
-			    <artifactId>jakarta.mail</artifactId>
+			    <artifactId>javax.mail</artifactId>
 			    <version>${mail.version}</version>
 			</artifactItem>
 		    </artifactItems>
--- a/parent-distrib/pom.xml
+++ b/parent-distrib/pom.xml
@@ -62,7 +62,7 @@
 			    <artifactItems>
 				<artifactItem>
 				    <groupId>com.sun.mail</groupId>
-				    <artifactId>jakarta.mail</artifactId>
+				    <artifactId>javax.mail</artifactId>
 				    <version>${mail.version}</version>
 				    <classifier>sources</classifier>
 				    <outputDirectory>
@@ -77,7 +77,7 @@
 		    <artifactItems>
 			<artifactItem>
 			    <groupId>com.sun.mail</groupId>
-			    <artifactId>jakarta.mail</artifactId>
+			    <artifactId>javax.mail</artifactId>
 			    <version>${mail.version}</version>
 			</artifactItem>
 		    </artifactItems>
